// [[Reason: TODO: Sourcemap seems missaligned]] ////

//// [tests/cases/compiler/declarationEmitExportAliasVisibiilityMarking.ts] ////

===================================================================
--- TSC	declarations
+++ DTE	declarations
@@ -1,9 +1,9 @@
 
 //// [Card.d.ts.map]
-{"version":3,"file":"Card.d.ts","sourceRoot":"","sources":["Card.ts"],"names":[],"mappings":"AAAA,OAAO,EAAE,IAAI,EAAE,IAAI,EAAE,MAAM,SAAS,CAAC;+BACf,IAAI,QAAQ,IAAI;UAC5B,IAAI;UACJ,IAAI;;AAFd,wBAGoB"}
+{"version":3,"file":"Card.d.ts","sourceRoot":"","sources":["Card.ts"],"names":[],"mappings":"AAAA,OAAO,EAAE,IAAI,EAAE,IAAI,EAAE,MAAM,SAAS,CAAC;yBACrB,IAAI,EAAE,IAAI,EAAE,IAAI,EAAE,IAAI,KAAG;IACrC,IAAI,EAAE,IAAI,CAAC;IACX,IAAI,EAAE,IAAI,CAAC;CACd;AAHD,wBAGoB"}
 
-//// https://sokra.github.io/source-map-visualization#base64,aW1wb3J0IHsgU3VpdCwgUmFuayB9IGZyb20gJy4vVHlwZXMnOw0KZGVjbGFyZSBjb25zdCBfZGVmYXVsdDogKHN1aXQ6IFN1aXQsIHJhbms6IFJhbmspID0+IHsNCiAgICBzdWl0OiBTdWl0Ow0KICAgIHJhbms6IFJhbms7DQp9Ow0KZXhwb3J0IGRlZmF1bHQgX2RlZmF1bHQ7DQovLyMgc291cmNlTWFwcGluZ1VSTD1DYXJkLmQudHMubWFw,eyJ2ZXJzaW9uIjozLCJmaWxlIjoiQ2FyZC5kLnRzIiwic291cmNlUm9vdCI6IiIsInNvdXJjZXMiOlsiQ2FyZC50cyJdLCJuYW1lcyI6W10sIm1hcHBpbmdzIjoiQUFBQSxPQUFPLEVBQUUsSUFBSSxFQUFFLElBQUksRUFBRSxNQUFNLFNBQVMsQ0FBQzsrQkFDZixJQUFJLFFBQVEsSUFBSTtVQUM1QixJQUFJO1VBQ0osSUFBSTs7QUFGZCx3QkFHb0IifQ==,aW1wb3J0IHsgU3VpdCwgUmFuayB9IGZyb20gJy4vVHlwZXMnOwpleHBvcnQgZGVmYXVsdCAoc3VpdDogU3VpdCwgcmFuazogUmFuayk6IHsKICAgIHN1aXQ6IFN1aXQ7CiAgICByYW5rOiBSYW5rOwp9ID0+ICh7c3VpdCwgcmFua30pOwo=
+//// https://sokra.github.io/source-map-visualization#base64,aW1wb3J0IHsgU3VpdCwgUmFuayB9IGZyb20gJy4vVHlwZXMnOw0KZGVjbGFyZSBjb25zdCBfZGVmYXVsdDogKHN1aXQ6IFN1aXQsIHJhbms6IFJhbmspID0+IHsNCiAgICBzdWl0OiBTdWl0Ow0KICAgIHJhbms6IFJhbms7DQp9Ow0KZXhwb3J0IGRlZmF1bHQgX2RlZmF1bHQ7DQovLyMgc291cmNlTWFwcGluZ1VSTD1DYXJkLmQudHMubWFw,eyJ2ZXJzaW9uIjozLCJmaWxlIjoiQ2FyZC5kLnRzIiwic291cmNlUm9vdCI6IiIsInNvdXJjZXMiOlsiQ2FyZC50cyJdLCJuYW1lcyI6W10sIm1hcHBpbmdzIjoiQUFBQSxPQUFPLEVBQUUsSUFBSSxFQUFFLElBQUksRUFBRSxNQUFNLFNBQVMsQ0FBQzt5QkFDckIsSUFBSSxFQUFFLElBQUksRUFBRSxJQUFJLEVBQUUsSUFBSSxLQUFHO0lBQ3JDLElBQUksRUFBRSxJQUFJLENBQUM7SUFDWCxJQUFJLEVBQUUsSUFBSSxDQUFDO0NBQ2Q7QUFIRCx3QkFHb0IifQ==,aW1wb3J0IHsgU3VpdCwgUmFuayB9IGZyb20gJy4vVHlwZXMnOwpleHBvcnQgZGVmYXVsdCAoc3VpdDogU3VpdCwgcmFuazogUmFuayk6IHsKICAgIHN1aXQ6IFN1aXQ7CiAgICByYW5rOiBSYW5rOwp9ID0+ICh7c3VpdCwgcmFua30pOwo=
 
 
 //// [Types.d.ts.map]
 {"version":3,"file":"Types.d.ts","sourceRoot":"","sources":["Types.ts"],"names":[],"mappings":"AAAA,KAAK,IAAI,GAAG,QAAQ,GAAG,QAAQ,GAAG,OAAO,GAAG,UAAU,CAAC;AACvD,KAAK,IAAI,GAAG,CAAC,GAAG,CAAC,GAAG,CAAC,GAAG,CAAC,GAAG,CAAC,GAAG,CAAC,GAAG,CAAC,GAAG,CAAC,GAAG,CAAC,GAAG,CAAC,GAAG,EAAE,GAAG,MAAM,GAAG,OAAO,GAAG,MAAM,CAAC;AACnF,OAAO,EAAE,IAAI,EAAE,IAAI,EAAE,CAAC"}
@@ -11,8 +11,8 @@
 //// https://sokra.github.io/source-map-visualization#base64,dHlwZSBTdWl0ID0gJ0hlYXJ0cycgfCAnU3BhZGVzJyB8ICdDbHVicycgfCAnRGlhbW9uZHMnOw0KdHlwZSBSYW5rID0gMCB8IDEgfCAyIHwgMyB8IDQgfCA1IHwgNiB8IDcgfCA4IHwgOSB8IDEwIHwgJ0phY2snIHwgJ1F1ZWVuJyB8ICdLaW5nJzsNCmV4cG9ydCB7IFN1aXQsIFJhbmsgfTsNCi8vIyBzb3VyY2VNYXBwaW5nVVJMPVR5cGVzLmQudHMubWFw,eyJ2ZXJzaW9uIjozLCJmaWxlIjoiVHlwZXMuZC50cyIsInNvdXJjZVJvb3QiOiIiLCJzb3VyY2VzIjpbIlR5cGVzLnRzIl0sIm5hbWVzIjpbXSwibWFwcGluZ3MiOiJBQUFBLEtBQUssSUFBSSxHQUFHLFFBQVEsR0FBRyxRQUFRLEdBQUcsT0FBTyxHQUFHLFVBQVUsQ0FBQztBQUN2RCxLQUFLLElBQUksR0FBRyxDQUFDLEdBQUcsQ0FBQyxHQUFHLENBQUMsR0FBRyxDQUFDLEdBQUcsQ0FBQyxHQUFHLENBQUMsR0FBRyxDQUFDLEdBQUcsQ0FBQyxHQUFHLENBQUMsR0FBRyxDQUFDLEdBQUcsRUFBRSxHQUFHLE1BQU0sR0FBRyxPQUFPLEdBQUcsTUFBTSxDQUFDO0FBQ25GLE9BQU8sRUFBRSxJQUFJLEVBQUUsSUFBSSxFQUFFLENBQUMifQ==,dHlwZSBTdWl0ID0gJ0hlYXJ0cycgfCAnU3BhZGVzJyB8ICdDbHVicycgfCAnRGlhbW9uZHMnOwp0eXBlIFJhbmsgPSAwIHwgMSB8IDIgfCAzIHwgNCB8IDUgfCA2IHwgNyB8IDggfCA5IHwgMTAgfCAnSmFjaycgfCAnUXVlZW4nIHwgJ0tpbmcnOwpleHBvcnQgeyBTdWl0LCBSYW5rIH07Cg==
 
 
 //// [index.d.ts.map]
-{"version":3,"file":"index.d.ts","sourceRoot":"","sources":["index.ts"],"names":[],"mappings":"AAAA,OAAO,EAAE,IAAI,EAAE,IAAI,EAAE,MAAM,SAAS,CAAC;AAErC,eAAO,IAAI,QAAQ,uBAAsB,IAAI,QAAQ,IAAI;UAC/C,IAAI;UACJ,IAAI;EAC6B,CAAC;AAC5C,OAAO,EAAE,IAAI,EAAE,IAAI,EAAE,MAAM,SAAS,CAAC"}
+{"version":3,"file":"index.d.ts","sourceRoot":"","sources":["index.ts"],"names":[],"mappings":"AAAA,OAAO,EAAE,IAAI,EAAE,IAAI,EAAE,MAAM,SAAS,CAAC;AAErC,eAAO,IAAI,QAAQ,QAAO,OAAO,CAAC,CAAC,IAAI,EAAE,IAAI,EAAE,IAAI,EAAE,IAAI,KAAK;IAC1D,IAAI,EAAE,IAAI,CAAC;IACX,IAAI,EAAE,IAAI,CAAC;CACd,CAA0C,CAAC;AAC5C,OAAO,EAAE,IAAI,EAAE,IAAI,EAAE,MAAM,SAAS,CAAC"}
 
-//// https://sokra.github.io/source-map-visualization#base64,aW1wb3J0IHsgU3VpdCwgUmFuayB9IGZyb20gJy4vVHlwZXMnOw0KZXhwb3J0IGRlY2xhcmUgbGV0IGxhenlDYXJkOiAoKSA9PiBQcm9taXNlPChzdWl0OiBTdWl0LCByYW5rOiBSYW5rKSA9PiB7DQogICAgc3VpdDogU3VpdDsNCiAgICByYW5rOiBSYW5rOw0KfT47DQpleHBvcnQgeyBTdWl0LCBSYW5rIH0gZnJvbSAnLi9UeXBlcyc7DQovLyMgc291cmNlTWFwcGluZ1VSTD1pbmRleC5kLnRzLm1hcA==,eyJ2ZXJzaW9uIjozLCJmaWxlIjoiaW5kZXguZC50cyIsInNvdXJjZVJvb3QiOiIiLCJzb3VyY2VzIjpbImluZGV4LnRzIl0sIm5hbWVzIjpbXSwibWFwcGluZ3MiOiJBQUFBLE9BQU8sRUFBRSxJQUFJLEVBQUUsSUFBSSxFQUFFLE1BQU0sU0FBUyxDQUFDO0FBRXJDLGVBQU8sSUFBSSxRQUFRLHVCQUFzQixJQUFJLFFBQVEsSUFBSTtVQUMvQyxJQUFJO1VBQ0osSUFBSTtFQUM2QixDQUFDO0FBQzVDLE9BQU8sRUFBRSxJQUFJLEVBQUUsSUFBSSxFQUFFLE1BQU0sU0FBUyxDQUFDIn0=,aW1wb3J0IHsgU3VpdCwgUmFuayB9IGZyb20gJy4vVHlwZXMnOwoKZXhwb3J0IGxldCBsYXp5Q2FyZCA9ICgpOiBQcm9taXNlPChzdWl0OiBTdWl0LCByYW5rOiBSYW5rKSA9PiB7CiAgICBzdWl0OiBTdWl0OwogICAgcmFuazogUmFuazsKfT4gPT4gaW1wb3J0KCcuL0NhcmQnKS50aGVuKGEgPT4gYS5kZWZhdWx0KTsKZXhwb3J0IHsgU3VpdCwgUmFuayB9IGZyb20gJy4vVHlwZXMnOwo=
+//// https://sokra.github.io/source-map-visualization#base64,aW1wb3J0IHsgU3VpdCwgUmFuayB9IGZyb20gJy4vVHlwZXMnOw0KZXhwb3J0IGRlY2xhcmUgbGV0IGxhenlDYXJkOiAoKSA9PiBQcm9taXNlPChzdWl0OiBTdWl0LCByYW5rOiBSYW5rKSA9PiB7DQogICAgc3VpdDogU3VpdDsNCiAgICByYW5rOiBSYW5rOw0KfT47DQpleHBvcnQgeyBTdWl0LCBSYW5rIH0gZnJvbSAnLi9UeXBlcyc7DQovLyMgc291cmNlTWFwcGluZ1VSTD1pbmRleC5kLnRzLm1hcA==,eyJ2ZXJzaW9uIjozLCJmaWxlIjoiaW5kZXguZC50cyIsInNvdXJjZVJvb3QiOiIiLCJzb3VyY2VzIjpbImluZGV4LnRzIl0sIm5hbWVzIjpbXSwibWFwcGluZ3MiOiJBQUFBLE9BQU8sRUFBRSxJQUFJLEVBQUUsSUFBSSxFQUFFLE1BQU0sU0FBUyxDQUFDO0FBRXJDLGVBQU8sSUFBSSxRQUFRLFFBQU8sT0FBTyxDQUFDLENBQUMsSUFBSSxFQUFFLElBQUksRUFBRSxJQUFJLEVBQUUsSUFBSSxLQUFLO0lBQzFELElBQUksRUFBRSxJQUFJLENBQUM7SUFDWCxJQUFJLEVBQUUsSUFBSSxDQUFDO0NBQ2QsQ0FBMEMsQ0FBQztBQUM1QyxPQUFPLEVBQUUsSUFBSSxFQUFFLElBQUksRUFBRSxNQUFNLFNBQVMsQ0FBQyJ9,aW1wb3J0IHsgU3VpdCwgUmFuayB9IGZyb20gJy4vVHlwZXMnOwoKZXhwb3J0IGxldCBsYXp5Q2FyZCA9ICgpOiBQcm9taXNlPChzdWl0OiBTdWl0LCByYW5rOiBSYW5rKSA9PiB7CiAgICBzdWl0OiBTdWl0OwogICAgcmFuazogUmFuazsKfT4gPT4gaW1wb3J0KCcuL0NhcmQnKS50aGVuKGEgPT4gYS5kZWZhdWx0KTsKZXhwb3J0IHsgU3VpdCwgUmFuayB9IGZyb20gJy4vVHlwZXMnOwo=
 
